erts/emulator/beam/jit/arm/predicates.tab

```diff
@@ -81,6 +81,10 @@ pred.never_fails(Bif) {
     return 0;
 }
 
+pred.consecutive_xregs(A1, A2) {
+    return A1.type == TAG_x && A2.type == TAG_x && A1.val + 1 == A2.val && A1.val >= 6;
```

Comment:
Nitpick: `A1.val >= 6` is a bit magical, perhaps add a comment?

Reply:
Done.